docs/source/contributing/ci.adoc

```diff
@@ -60,3 +60,9 @@ link:https://ci.centos.org/job/minishift-release/[release]
 |link:https://travis-ci.org/minishift/minishift/branches[master], link:https://travis-ci.org/minishift/minishift/pull_requests[pull requests]
 |link:https://github.com/minishift/minishift/blob/master/.travis.yml[.travis.yml]
 |===
+
+[[nightly-job]]
+== Nightly Job
+
+A link:https://ci.centos.org/job/minishift-nightly[nightly job] has been configured to be run daily at midnight (2:20 AM UTC) which executes all tests against the `master` branch from the link:https://github.com/minishift/minishift[minishift] repo.
```

Comment:
s/A nightly job has been configured to be run/The nightly test  job runs daily/. 

Suggestion:
The nightly test  job runs daily at 2:20AM UTC). It executes all tests against the master branch of the Minishift github repository.

Reply:
Done with this change here https://github.com/minishift/minishift-ci-jobs/commit/8f964f783ab2d05412ae4a99be9f421c603f754f

Reply:
there is still a disconnect here. The config is for midnight even if you changed from @midnight to 0 0 * * *. So what's going on? What timezone is set on the slave? What's the timezone on the master? Do we need to set the proper time zone on the slave as part of initialization? Do we need to enable ntp. IMO it is not ok, to just say  "it starts 2:20AM UTC" even though midnight is configured. It is also not ok to make this assumption based on a single run like this. Who says this time displayed is directly related to the trigger? 

Let's find out what's going on here and do this properly.

Reply:
@hferentschik Confirmed with @bstinsonmhk that default time zone `UTC` is set on all CI machines. 

> Do we need to set the proper time zone on the slave as part of initialization?

Setting time zone seems starting jenkins server altogether which might not be feasible here https://wiki.jenkins.io/display/JENKINS/Change+time+zone.

> Who says this time displayed is directly related to the trigger?

With `0 0 * * *` should start the job at sharp 00:00 AM as `@midnight` in terms of Jenkins world means run job in between 00:00 AM to 2:59 AM. 
See https://stackoverflow.com/a/17204211/1120530.